Sources/FingerprintJS/SystemControl/RawPointerConvertible.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+protocol RawPointerConvertible {
+    static func loadValue(_ from: UnsafeMutableRawPointer, bytes: size_t) -> Self
+    static func withMemoryInput(
```

Comment:
How about renaming this method to `withRawMemory`? Since it calls the supplied `body` closure with a pointer to block of allocated memory that is in uninitialized state.